README.md

```diff
@@ -100,6 +100,44 @@ in /etc/repospanner.
 This client will automatically revert to plain git if it determines the repo
 that is being pushed to is not a repospanner repository.
 
+
+Operating Repospanner cluster
+-----------------------------
```

Comment:
Could you rebase your PR on top of `master` please?
README.md has been switched from Setext to atx style of headers, so could you update this to use atx?